src/codeu/chat/server/Server.java

```diff
@@ -42,245 +42,251 @@
 
 public final class Server {
 
-  private interface Command {
-    void onMessage(InputStream in, OutputStream out) throws IOException;
-  }
+	private interface Command {
+		void onMessage(InputStream in, OutputStream out) throws IOException;
+	}
+
+	private static final Logger.Log LOG = Logger.newLog(Server.class);
 
-  private static final Logger.Log LOG = Logger.newLog(Server.class);
+	private static final int RELAY_REFRESH_MS = 5000; // 5 seconds
```

Comment:
You don't have to use the formatter, just change the tabs back into spaces. I suggested it because I thought formatter plugin would integrate well with IDEs, but maybe [configuring your IDE](https://stackoverflow.com/questions/407929/how-do-i-change-eclipse-to-use-spaces-instead-of-tabs) would be easier.

I think you might be using the command-line fomatter. In that case, note that if I run it with `-h`, it says:  

    $ java -jar ~/bin/google-java-format-1.3-all-deps.jar -h

    Usage: google-java-format [options] file(s)
    
    Options:
      -i, -r, -replace, --replace
        Send formatted output back to files, not stdout.  
    ...

So the default behavior sends output to stdout, and you'll need one of the above flags to replace the file.